cogboard-webapp/src/components/widgets/types/AemBundleInfoWidget.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { object } from 'prop-types';
+
+import { Caption } from '../../styled';
+import { capitalize } from '../../../utils/common';
+import { Typography } from '@material-ui/core';
+import { PopoverWithControls } from '../../PopoverWithControls';
+
+const bundleArrayToComponents = array => (
+  <>
+    {array.map(object => (
+      <Typography variant="body2">
+        {object['name']} - {object['state']} (id: {object['id']}, symbolicName:{' '}
+        {object['symbolicName']})
+      </Typography>
+    ))}
```

Comment:
This could use some links, so you'd be able to click inside popover and see what went wrong at the instance